latinize/latinize.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for latinize 0.2.0
+// Project: https://github.com/dundalek/latinize
+// Definitions by: Giedrius Grabauskas <https://github.com/GiedriusGrabauskas>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace Latinize {
+    export interface ILatinize {
```

Comment:
http://www.typescriptlang.org/docs/handbook/writing-declaration-files.html

> Naming Conventions
In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.